Fitness/Views/ProRoutineView.swift

```diff
@@ -0,0 +1,156 @@
+        titleLabel.textAlignment = .left
+        titleLabel.text = routine.title
+        titleLabel.sizeToFit()
+        self.addSubview(titleLabel)
+        
+        routineTypeImage = UIImageView()
+        routineTypeImage.contentMode = .scaleAspectFit
+        routineTypeImage.clipsToBounds = true
+        self.addSubview(routineTypeImage)
+        
+        routineTypeLabel = UILabel()
+        routineTypeLabel.textColor = UIColor.colorFromCode(0x555555)
+        routineTypeLabel.font = ._12MontserratRegular
+        routineTypeLabel.textAlignment = .center
+        routineTypeLabel.sizeToFit()
+        self.addSubview(routineTypeLabel)
+        
+        switch routine.routineType {
+        case .cardio:
+            routineTypeImage.image = #imageLiteral(resourceName: "small cardio icon.png")
+            routineTypeLabel.text = "Cardio"
+        case .mindfullness:
+            routineTypeImage.image = #imageLiteral(resourceName: "smaller mindfulness icon.png")
+            routineTypeLabel.text = "Mindfulness"
+        case .strength:
+            routineTypeImage.image = #imageLiteral(resourceName: "small strength icon.png")
+            routineTypeLabel.text = "Strength"
+        case .other:
+            routineTypeImage.image = #imageLiteral(resourceName: "smaller mindfulness icon.png")
+            routineTypeLabel.text = "Other"
+        }
+        
+        addButton = UIButton()
+        addButton.setBackgroundImage(#imageLiteral(resourceName: "add habit.png"), for: .normal)
+//        self.addSubview(addButton)
+        // Commenting out the add button until more functionality to save and display favourited routines is implemented
+        
+        stepsTextView = UITextView()
+        stepsTextView.textColor = UIColor.fitnessBlack
+        stepsTextView.font = ._14MontserratLight
+        stepsTextView.isScrollEnabled = false
+        stepsTextView.textAlignment = .left
+        stepsTextView.isSelectable = false
+        stepsTextView.text = routine.text
+        stepsTextView.sizeToFit()
+        
+        let style = NSMutableParagraphStyle()
+        style.lineSpacing = 5
+        let attributes = [NSAttributedString.Key.paragraphStyle : style, NSAttributedString.Key.font : UIFont._14MontserratLight]
+        stepsTextView.attributedText = NSAttributedString(string: routine.text, attributes: attributes)
+        
+        self.addSubview(stepsTextView)
+        
+        addRoundedCorners(radius: 5)
+        applySketchShadow(color: UIColor.fitnessGreen, alpha: 0.03, y: 11, blur: 14, spread: -10)
+        
+    }
+    
+    func addRoundedCorners(radius: CGFloat) {
+        layer.cornerRadius = radius
+        layer.masksToBounds = false
+        layer.borderColor = UIColor.fitnessMutedGreen.cgColor
+        layer.borderWidth = 1
+    }
+    
+    func applySketchShadow(color: UIColor = .black, alpha: Float = 0.5, x: CGFloat = 0, y: CGFloat = 2, blur: CGFloat = 4, spread: CGFloat = 0)
```

Comment:
pretty certain that this is over 120 or the typical line length, this should be split across multiple lines

Reply:
Will do